tests/automation_framework/tests/work_order_tests/test_submit.py

```diff
@@ -1815,3 +1815,443 @@ def test_work_order_submit_requesterNonce_param_empty(self):
+            constants.work_order_input_file,
+            "workordersubmit_indata_outdata.json")
+
+        err_cd = \
+            self.test_obj.setup_and_build_request_wo_submit(
+                read_json(request_file))
+
+        submit_response = submit_request(
+            self.test_obj.uri_client,
+            self.test_obj.build_request_output['request_obj'],
+            constants.wo_submit_output_json_file_name,
+            read_json(request_file))
+
+        result_response = self.test_obj.getresult(
+            self.test_obj.build_request_output['request_obj'])
+        assert (
+                verify_test(
+                    result_response, 0,
+                    self.test_obj.build_request_output['pre_test_output'],
+                    self.test_obj.build_request_output['action_obj'])
+                is TestStep.SUCCESS.value)
+        logger.info('\t\t!!! Test completed !!!\n\n')
+
+    @pytest.mark.work_order_submit
+    @pytest.mark.test_workordersubmit_workorderId_remove
+    @pytest.mark.sdk
+    def test_workordersubmit_workorderId_remove(self):
+        request_file = os.path.join(
+            constants.work_order_input_file,
+            "workordersubmit_workorderId_remove.json")
+
+        err_cd = \
+            self.test_obj.setup_and_build_request_wo_submit(
+                read_json(request_file))
+
+        submit_response = submit_request(
+            self.test_obj.uri_client,
+            self.test_obj.build_request_output['request_obj'],
+            constants.wo_submit_output_json_file_name,
+            read_json(request_file))        
+        assert (
+                  check_negative_test_responses(
+                   submit_response,
+                   "Invalid data format for work order id")
+                  is TestStep.SUCCESS.value)
+        logger.info('\t\t!!! Test completed !!!\n\n')
+
+
+    @pytest.mark.work_order_submit
+    @pytest.mark.test_workordersubmit_sessionkeyiv_allspecialchar
+    @pytest.mark.listener
+    def test_workordersubmit_sessionkeyiv_allspecialchar(self):
+        request_file = os.path.join(
+            constants.work_order_input_file,
+            "workordersubmit_sessionkeyiv_allspecialchar.json")
+
+        err_cd = \
+            self.test_obj.setup_and_build_request_wo_submit(
+                read_json(request_file))
+
+        submit_response = submit_request(
+            self.test_obj.uri_client,
+            self.test_obj.build_request_output['request_obj'],
+            constants.wo_submit_output_json_file_name,
+            read_json(request_file))
+
+        assert (
+                  check_negative_test_responses(
+                   submit_response,
+                   "Invalid data format for session key iv")
+                  is TestStep.SUCCESS.value)
+        logger.info('\t\t!!! Test completed !!!\n\n')
+
+    @pytest.mark.work_order_submit
+    @pytest.mark.test_workordersubmit_requesterId_variouslengthhex
```

Comment:
Why is the tests passing when you are passing unexpected value? Is this expected?